k2/csrc/rm_epsilon.cu

```diff
@@ -586,7 +586,11 @@ void ComputeEpsilonClosureOneIter(FsaVec &epsilon_fsa, FsaVec *closure_fsa,
 
 void RemoveEpsilonsIterativeTropical(FsaVec &src_fsa, FsaVec *dest_fsa,
```

Comment:
@danpovey , I just fell we do too many things in this function and the code may be hard to read for others. If you think so, I'll split it into a few functions, e.g. one to compute `epsilon_closure_mapped`, one to compute `epsilon_num_foll_arcs` and `epsilon_pre_renumbering`, one to compute `combined_foll`, one to compute `combined_prec`, one to get the result, etc. Of course not so urgent, I'll try to add random tests first.

Reply:
ok with me...